website/content/api-docs/system/internal-counters.mdx

```diff
@@ -347,6 +349,7 @@ This endpoint was added in Vault 1.6.
 
 - `start_time` `(string, optional)` - An RFC3339 timestamp or Unix epoch time. Specifies the start of the
   period for which client counts will be reported. If no start time is specified, the billing start date will be used.
+  The [billing start date](/docs/concepts/billing-start-date) automatically rolls over to the latest billing year at the end of the last cycle.
```

Comment:
Did not mention what auto-rolls means here because I linked billing-start-date concepts doc.